soc/arm/st_stm32/stm32f0/Kconfig.soc

```diff
@@ -34,6 +34,9 @@ config SOC_STM32F070XB
 config SOC_STM32F072XB
 	bool "STM32F072XB"
 
+config SOC_STM32F072X8
```

Comment:
Nitpick: Add that just before F072XB, to keep them in "alphabetical" order

Reply:
fixed it!